include/mpd/recv.h

```diff
@@ -46,6 +46,14 @@ struct mpd_connection;
 extern "C" {
 #endif
 
+/**
+ * Reads the binary data from the server.  Returns NULL on error.
+ *
+ * The caller must free the allocated memory.
+ */
+char * 
+mpd_recv_binary(struct mpd_connection *connection, const unsigned binary);
```

Comment:
Another pointless `const`, and API documentation missing.

Reply:
I use the const keyword to signal, that this function doesn't alter this value.

Reply:
Why would the caller care if the function changes the register value containing that parameter?
And having `const` here doesn't even imply that the function doesn't alter this value...